### DISCUSSION TITLE
GUI for Mac?

### DISCUSSION THREAD

**Discussion by rowild** [openMVG/openMVG#2272] [Q&A] [answered]
Being new to openMVG, I understand that this is a library that is supposed to be use on the command line or as a 3rd party tool. But in the context of the Linux installation, the word GUI is dropped here and there (and I find some Youtube videos where a GUI is used). I was wondering if it was possible to build a GUI on a Mac (M2), too?

**Comment by pmoulon** [comment, marked as answer]
Hi. The GUI as today does not perform reconstruction, they are just helpers for some steps (rigid registration with control points)
